cpp/tests/data/pydata/District/immunity_population.txt

```diff
@@ -0,0 +1,3 @@
+0.04 0.04 0.075 0.08 0.035 0.01
+0.61 0.61 0.62 0.62 0.58 0.41
+0.35 0.35 0.305 0.3 0.385 0.58
```

Comment:
A better design is probably to use a matrix to pass the values to the function that uses them. This still allows using a file like this by reading it into a matrix, then passing it to that function. But the file is not *required* to be there (for tests etc.).